src/components/FeaturedServices/FeaturedServices.module.scss

```diff
@@ -0,0 +1,35 @@
+@import 'global.scss';
```

Comment:
Let's delete this and the summary.js in the `components` directory, it's been moved to the `home` directory, and you must've not pullen from master